app/assets/stylesheets/galleries.scss

```diff
@@ -73,3 +73,20 @@
   }
 }
 .loading-icon { width: 16px; height: 16px; }
+
+/* Various tables with icons in previously styled with .green */
+.icons-box {
+  background-color: #B0BBA9;
+}
+
+/* Icon-related text previously styled with .green */
+.icon-text {
```

Comment:
This is for, specifically, `credit` on an icon's page, and for the "No galleries yet" box, I think? What does the credit thing look like – I don't think these are semantically similar?

Reply:
I think I would put the "No galleries yet" thing as just a piece of plain text in a `.icons-box` – it feels to me like it's the same element, semantically, as the thing that lists icons and galleries, it just doesn't have galleries in.

The credit thing should probably be styled together with the icon, if it can be? I actually have not looked at that page yet, uh, apologies for the comments that are partially guessing.

Reply:
it is those two things! I can put the 'no galleries yet' in an .icons-box yeah. and. uh. idk about the credit? it was styled separately when i found it.

Reply:
Maybe check for uses of `.credit` and if you don't see it anywhere, change this selector to `.credit`? If you do see it elsewhere, I'd rename this to some third thing, which makes it clear it only applies to `credit` in that one location wherever it was, since `icon-text` sounds like it could be talking about the keywords below icons, say, when you're adding existing icons to a gallery.